lib/errors.js

```diff
@@ -26,5 +26,9 @@ module.exports = {
   AVV_ERR_READY_TIMEOUT: createError(
     'AVV_ERR_READY_TIMEOUT',
     "Plugin did not start in time: '%s'. You may have forgotten to call 'done' function or to resolve a Promise"
+  ),
+  AVV_ERR_PLUGIN_EXEC_TIMEOUT: createError(
+    'AVV_ERR_PLUGIN_EXEC_TIMEOUT',
```

Comment:
The change is sound to me but they are really similar by now. Shall we change the message to differentiate?

Reply:
see comment above ;)